src-plugins/itkFilters/itkMorphologicalFiltersProcessBase.h

```diff
@@ -24,10 +24,18 @@ class ITKFILTERSPLUGIN_EXPORT itkMorphologicalFiltersProcessBase : public itkFil
 {
     Q_OBJECT
 public:
+    enum KernelShape
+    {
+        BallKernel,
+        CrossKernel,
+        BoxKernel,
```

Comment:
I don't usually do it but it's valid C++ syntax anyways (some devs do it because it's easier to add, remove or change the order of elements)